src/pages/iou/request/step/IOURequestStepScan/LocationPermission/index.ts

```diff
@@ -0,0 +1,37 @@
+import {RESULTS} from 'react-native-permissions';
+import type {PermissionStatus} from 'react-native-permissions';
+
+function requestLocationPermission(): Promise<PermissionStatus> {
+    return new Promise((resolve) => {
+        if (navigator.geolocation) {
+            navigator.geolocation.getCurrentPosition(
+                () => resolve(RESULTS.GRANTED),
+                (error) => resolve(error.TIMEOUT || error.POSITION_UNAVAILABLE ? RESULTS.BLOCKED : RESULTS.DENIED),
+                {
+                    timeout: 5000,
```

Comment:
COuld you please make this a const, like [this](https://github.com/Expensify/App/blob/6c5d186c3179251d83f1abb437a3e4616a1191c0/src/CONST.ts#L4737).

Reply:
Done